metricbeat/helper/windows/pdh/pdh_query_windows.go

```diff
@@ -218,6 +234,7 @@ func (q *Query) ExpandWildCardPath(wildCardPath string) ([]string, error) {
 		if err == PDH_MORE_DATA {
 			expdPaths, err = PdhExpandCounterPath(utfPath)
 		}
+		expdPaths, err = PdhExpandCounterPath(utfPath)
```

Comment:
is this intended? IT would look like you would call `PdhExpandCounterPath` many times here (both in case of error and no error)

Reply:
it was not intended, just for testing purposes, refactored the func